apps/extension/src/pages/earn/withdraw/amount.tsx

```diff
@@ -0,0 +1,456 @@
+      if (!nobleEarnAmountConfig.currency) {
+        throw new Error("Withdraw currency not set");
+      }
+
+      if (
+        nobleEarnAmountConfig.uiProperties.loadingState === "loading-block" ||
+        nobleEarnAmountConfig.uiProperties.error != null
+      ) {
+        throw new Error("Not ready to simulate tx");
+      }
+
+      return account.noble.makeSwapTx(
+        "noble-earn-withdraw",
+        nobleEarnAmountConfig.amount[0].toDec().toString(),
+        currency,
+        nobleEarnAmountConfig.minOutAmount.toDec().toString(),
+        outCurrency,
+        [
+          {
+            poolId: poolForWithdraw?.id.toString() ?? "",
+            denomTo: outCurrency.coinMinimalDenom,
+          },
+        ]
+      );
+    }
+  );
+
+  const txConfigsValidate = useTxConfigsValidate({
+    senderConfig,
+    amountConfig: nobleEarnAmountConfig,
+    gasSimulator,
+  });
+
+  return (
+    <HeaderLayout
+      title={""} // No title for this page
+      displayFlex={true}
+      fixedMinHeight={true}
+      left={
+        <BackButton
+          {...(isConfirmView
+            ? {
+                onClick: () => {
+                  setIsConfirmView(false);
+                },
+              }
+            : {})}
+        />
+      }
+      bottomButtons={[
+        {
+          disabled: txConfigsValidate.interactionBlocked,
+          text: intl.formatMessage({ id: "button.next" }),
+          color: "primary",
+          size: "large",
+          type: "submit",
+          isLoading:
+            accountStore.getAccount(chainId).isSendingMsg === "ibcTransfer",
+        },
+      ]}
+      onSubmit={async (e) => {
+        e.preventDefault();
+
+        if (!isConfirmView) {
+          setIsConfirmView(true);
+        } else {
+          if (!poolForWithdraw) {
+            throw new Error("No pool for withdraw");
+          }
+
+          try {
+            const tx = account.noble.makeSwapTx(
+              "noble-earn-withdraw",
+              nobleEarnAmountConfig.amount[0].toDec().toString(),
+              currency,
+              nobleEarnAmountConfig.minOutAmount.toDec().toString(),
+              outCurrency,
+              [
+                {
+                  poolId: poolForWithdraw.id.toString(),
+                  denomTo: outCurrency.coinMinimalDenom,
+                },
+              ]
+            );
+
+            await tx.send(feeConfig.toStdFee(), undefined, undefined, {
+              onBroadcasted: (_txHash) => {
+                navigate("/tx-result/pending");
+
+                // TODO: Log analytics
+              },
+              onFulfill: (tx: any) => {
+                if (tx.code != null && tx.code !== 0) {
+                  console.log(tx.log ?? tx.raw_log);
+                  navigate("/tx-result/failed");
+
+                  return;
+                }
+
+                navigate("/tx-result/success");
+              },
+            });
+          } catch (e) {
+            console.error(e);
```

Comment:
유저가 서명을 취소한 경우에도 fail 화면으로 가게 되어서 예외처리 해줘야할 것 같아요.
```suggestion
            if (e?.message === "Request rejected") {
              setIsConfirmView(true);
              return;
            }
            console.error(e);
```